Market-Trading-System/src/main/java/DAL/ItemDTO.java

```diff
@@ -1,49 +1,58 @@
 package DAL;
 
 import DomainLayer.Market.Util.DataItem;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-import java.util.Date;
+import java.util.List;
 
 public class ItemDTO implements DataItem <Long>, Comparable {
+    @JsonProperty("itemId")
+    private final long itemId;
+
+    @JsonProperty("itemName")
+    private final String itemName;
+
+    @JsonProperty("quantity")
+    private final int quantity;
+
+    @JsonProperty("storeId")
     private final long storeId;
+
+    @JsonProperty("totalPrice")
     private final double totalPrice;
-    private final int quantity;
-    private final String itemName;
-    private final long itemId;
 
-    public ItemDTO(long itemId, String itemName, int quantity, long storeId, double totalPrice){
+
+    public ItemDTO(long itemId, String itemName, int quantity, long storeId, double totalPrice) {
```

Comment:
Item has also description and category ? We don't want to show that ?

Reply:
I will create a new issue for this. 
I also suggest removing the ID field so it won't be accessible to the client. 
What do you think?

Reply:
I think it ok but what about the key for dB storage?